doc/architecture/environmental_effect_model.adoc

```diff
@@ -1,10 +1,9 @@
 = Environmental effect model
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-Et malesuada fames ac turpis egestas.
-Mauris pharetra et ultrices neque ornare aenean euismod elementum nisi.
-Nulla facilisi morbi tempus iaculis urna id. Justo eget magna fermentum iaculis.
-Sed augue lacus viverra vitae congue eu. Velit ut tortor pretium viverra suspendisse.
-Amet commodo nulla facilisi nullam vehicula ipsum a.
-Nibh nisl condimentum id venenatis a.
-Diam vel quam elementum pulvinar etiam non quam lacus suspendisse.
\ No newline at end of file
+Environmental effect models consume sensor-view data and produce sensor-view data.
```

Comment:
As I understood, SensorData is produced (@kmeids , @pmai ?)

Reply:
As discussed in CCB on 07/07/2021: sensor-view data is correct.